.github/tests/upstream-authority-cert-manager/values.yaml

```diff
@@ -1,5 +1,5 @@
 spire-server:
   upstreamAuthority:
-    certManager:
+    cert-manager:
```

Comment:
Why are we swithcing this from camelCasing?

I prefer to continue with the camelCasing convention in the Helm API as opposed to mixing in the spire plugin naming on the API which doesn't make it very consistent.

Reply:
See https://github.com/spiffe/helm-charts/pull/227#discussion_r1173020275